test/bidi.spec.mjs

```diff
@@ -133,24 +133,24 @@ describe('bidi.js', function() {
             });
         });
 
-        describe('thai contexts', () => {
+        describe('thai contexts', function () {
             it('should match thai words in a given text', () => {
                 const tokenizer = bidiThai.tokenizer;
-                tokenizer.tokenize('The king said: เป็นคนใจดีสำหรับทุกคน because ความรักคือทุกสิ่ง');
+                bidiThai.processText('The king said: เป็นคนใจดีสำหรับทุกคน because ความรักคือทุกสิ่ง');
                 const ranges = tokenizer.getContextRanges('thaiWord');
                 const words = ranges.map(range => tokenizer.rangeToText(range));
                 assert.deepEqual(words, ['เป็นคนใจดีสำหรับทุกคน', 'ความรักคือทุกสิ่ง']);
             });
         });
     });
 
-    describe('noto emoji with ccmp', () => {
+    describe('noto emoji with ccmp', function () {
         let notoEmojiFont;
-        before(()=> {
+        beforeEach(()=> {
```

Comment:
same as above: Why do we need to load/parse the font before each test? If it's not being modified and therefore interfering with following tests, this just adds overhead and increases testing runtime.

Reply:
There was a bug here
https://github.com/opentypejs/opentype.js/blob/master/test/path.js#L158

where `global.document` were created at init, but deleted after each test (which broke some of them)